game/hackmon.cc

```diff
@@ -0,0 +1,16 @@
+#include <vector>
+#include <string>
```

Comment:
Looks like you don't need string or map anymore

Reply:
why wouldn't I need string?